install.sh

```diff
@@ -126,3 +126,7 @@ ${GREEN}
 ######
 ${RESET}
 "
+
+if ! command -v ddev && [ ${OS} = "MINGW64_NT-10.0" ] ; then
```

Comment:
Should the output of `command -v ddev` be wrapped in `> /dev/null` like the other commands sothat it doesn't print out "/c/Program Files/ddev/ddev" or "/usr/local/bin/ddev"?

Reply:
Yep. So many things.